src/m_input.c

```diff
@@ -304,7 +304,22 @@ static const struct
   { KEY_INS,        "insert" },
   { KEY_PAUSE,      "pause" },
   { KEY_DEL,        "del" },
-  { KEY_PRTSCR,     "prtscr" }
+  { KEY_PRTSCR,     "prtscr" },
+  { KEYP_0,         "num0" },
```

Comment:
Or `kp0`? Or `keyp0`? Or `keypad0`?

Reply:
I like the current `KEYP_*` scheme.

Reply:
I was asking about the human readable string. 😉

Reply:
I think `num*` is good enough. I doubt that many users edit the config manually.

Reply:
Alright. I was unsure, because this is also the string presented in the "key bindings" menu.

Reply:
Just tested Quake, GZDoom and Eternity - no one supports num keys, interesting... Anyway, I think `num*` is good for menus too.